src/components/blipModal/index.js

```diff
@@ -0,0 +1,87 @@
+import { ModalContainer } from './ModalContainer'
+
+export const MODAL_DEFAULTS = {
+  onOpen: () => {},
+  onClose: () => {},
+  onOk: () => {},
```

Comment:
Would it be better if it was "confirm" instead of "ok"?

Reply:
Done!